docs/sources/panels-visualizations/visualizations/bar-chart/index.md

```diff
@@ -49,14 +49,18 @@ Use these options to refine your visualization.
 - **Horizontal** - Will make the X axis the category axis.
 - **Vertical** - Will make the Y axis the category axis.
 
-### Rotate bar labels
+### Rotate X tick labels
```

Comment:
```suggestion
### Rotate x-axis tick labels
```

Reply:
If "x-tick" is the label in the UI, we should file a UX bug to get it changed to "x-axis".  X Tick may be the option name in the code, but it should not be the label presented to users.

Reply:
Yes, the UI shows `X tick`. I will open a UI bug for this. I assume the process is the one found in creating a new issue through github.